Assets/00_Content/Scripts/Menu/MenuController.cs

```diff
@@ -39,14 +44,21 @@ public class MenuController : MonoBehaviour {
 			SceneLoadManager.Instance.LoadTutorial();
 		}
 
-		public void GoToSettings() {
-			mainMenuPanel.SetActive(false);
-			settingsPanel.SetActive(true);
+		public void ShowPanel(GameObject panel) {
+			panels.Peek().SetActive(false);
+			panel.SetActive(true);
+			panels.Push(panel);
+
+			if (panels.Count > 1)
+				backButton.SetActive(true);
 		}
 
-		public void ReturnToMainMenu() {
-			settingsPanel.SetActive(false);
-			mainMenuPanel.SetActive(true);
+		public void ClosePanel() {
```

Comment:
Might want to make sure you're not popping off the last panel of the stack here.

Reply:
Good catch